RadixWallet/Features/TransactionReviewFeature/TransactionReview.swift

```diff
@@ -1115,6 +777,21 @@ extension TransactionReview {
 			}
 
 			public typealias NonFungible = OnLedgerEntity.NonFungibleToken
+
+			public struct PoolUnit: Sendable, Hashable {
+				public let poolName: String
+				public let resources: [Resource]
+				public var guarantee: TransactionClient.Guarantee?
```

Comment:
How do we handle guarantees for poolUnits, it just works as for any fungible resource?